pom.xml

```diff
@@ -12,16 +11,69 @@
     <description>A microservice for storing associations between user IDs and DIDs for applications that want to use DID-Auth</description>
 
     <properties>
-        <swagger2markup.version>1.3.1</swagger2markup.version>
+        <java.version>11</java.version>
         <asciidoc.input.directory>${project.basedir}/src/docs/asciidoc</asciidoc.input.directory>
         <swagger.output.dir>${project.basedir}/src/main/resources/swagger</swagger.output.dir>
         <asciidoc.snippet.directory>${project.build.directory}/asciidoc/snippets</asciidoc.snippet.directory>
         <asciidoc.generated.directory>${project.build.directory}/asciidoc/generated</asciidoc.generated.directory>
         <swagger.input>${swagger.output.dir}/swagger.sdk.json</swagger.input>
+
+
+        <!-- Docker settings -->
+        <docker.registry.push>hostname</docker.registry.push> <!-- Create <server> section in .m2/chainSettings.xml with this registry
+                                                                    as id to authenticate to the private registry -->
+        <docker.registry.push.secret-name>registry-secret</docker.registry.push.secret-name>
+        <docker.registry.pull>docker.io</docker.registry.pull> <!-- Create <server> section in .m2/chainSettings.xml with this registry as id to authenticate to the
+                                                                      private registry, or do a docker login -u <user> manually -->
+
+        <fabric8.generator.from>fabric8/java-alpine-openjdk11-jre</fabric8.generator.from>
+        <docker.repository>${fabric8.label.provider}/${project.artifactId}</docker.repository>
+        <docker.version-tag>latest</docker.version-tag>
+
+        <!-- AJP mode -->
```

Comment:
Please mention in the readme the possibility of AJP and HTTP (and the difference)

Reply:
Fixed in upcomming commit